.pre-commit-config.yaml

```diff
@@ -13,7 +13,7 @@ repos:
         # supported by your project here, or alternatively use
         # pre-commit's default_language_version, see
         # https://pre-commit.com/#top_level-default_language_version
-        language_version: python3.8
+        language_version: python3.10
```

Comment:
The changes in this ticket required jumping to python 3.10?

Reply:
No, it's just something that should have been done earlier that had gone unnoticed until now (I think it only matters when someone does `pre-commit install`).